.github/workflows/dev-deploy.yaml

```diff
@@ -17,5 +19,7 @@ jobs:
         run: vercel pull --yes --environment=production --token=${{ secrets.DEV_VERCEL_TOKEN }}
       - name: Build Project Artifacts
         run: vercel build --prod --token=${{ secrets.DEV_VERCEL_TOKEN }}
+      - name: Run Migrations
+        run: npm run migrations
```

Comment:
What do you think, is it possible to also add to GitHub actions manual jobs for running migration up and down in case of some issues?

Reply:
that depends on actual use cases, i think there is ability to have manually triggered jobs

but that is waaaay out of initial scope :)